dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils.java

```diff
@@ -120,41 +120,50 @@ private static String jsonifyGCEvent(int maxFieldLength, String wlpUserDir,
         ArrayList<Pair> pairs = genData.getPairs();
         KeyValuePair kvp = null;
         String key = null;
-        String value = null;
 
         StringBuilder sb = CollectorJsonHelpers.startGCJson(hostName, wlpUserDir, serverName);
+        try {
```

Comment:
the new try / catch blocks are no longer needed in this and CollectorJsonUtils1_1.